setup.sh

```diff
@@ -20,7 +20,7 @@ GYM_CODE_SIGNING_IDENTITY="" #Code Sign Identitify
 FL_HOCKEY_TEAMS="" #Hockey ID (number not name)
 FL_HOCKEY_NOTIFY= #Email team when new build avialable? 0 = No, 1 = Yes
 ICON_OVERLAY_ASSETS_BUNDLE="" #Path to .xcassets
-TAB_USE_TIME_FOR_BUILD_NUMBER= #Use Time and date for build number or BUILD_NUMBER environment variable (created by jenkis or team city) true = use time, false = use BUILD_NUMBER
+TAB_USE_TIME_FOR_BUILD_NUMBER=true #Use Time and date for build number or BUILD_NUMBER environment variable (created by jenkis or team city) true = use time, false = use BUILD_NUMBER
```

Comment:
Do we want to use date+time for build number by default when setup?

Reply:
I personally think its the most sensible thing since it ensures build numbers are always greater every build, regardless of where it's being sent to

Reply:
that's true